# Modulo 5
### Ejercicio 1: Utilizar operadores aritméticos

Python proporciona operadores aritméticos comunes para que puedas realizar operaciones matemáticas en tu código. Estos incluyen las cuatro operaciones principales de suma, resta, multiplicación y división.

Exploremos cómo podemos crear un programa que pueda calcular la distancia entre dos planetas. Comenzaremos usando dos distancias de planetas: Tierra (149.597.870 km) y Júpiter (778.547.200 km).

In [1]:
# Crear variables para almacenar las instancias

Tierra = 149597870
Jupiter = 778547200

### Realizar la operacion
Con los valores obtenidos, es el momento de añadir el código para realizar la operación. Restarás el primer planeta del segundo para determinar la distancia en kilómetros. A continuación, puedes convertir la distancia del kilómetro en millas multiplicándola por 0.621.

In [4]:
# Calcular distancia entre planetas
Distancia = Tierra - Jupiter
if (Distancia < 0):
    Distancia *= -1

628949330 390577533.93


### Prueba tu proyecto
Con el código creado, ejecuta el notebook para obtener el resultado. Deberías recibir lo siguiente:

628949330
390577534

In [7]:
print("La distancia entre los planetas en kilometros es: "+ str(Distancia))
print("La distancia entre los planetas en millas es: "+ str(Distancia*0.621))

La distancia entre los planetas en kilometros es: 628949330
La distancia entre los planetas en millas es: 390577533.93


# Ejercicio 2: convierte cadenas en números y usa valores absolutos
### Crear una aplicación para trabajar con números y entrada de usuario
Con frecuencia, deberás convertir los valores de cadena en números para realizar correctamente diferentes operaciones o determinar el valor absoluto de un número.

Para crear nuestra aplicación, queremos leer la distancia del sol para dos planetas, y luego mostrar la distancia entre los planetas. Haremos esto usando input para leer los valores, int para convertir a entero y luego abs para convertir el resultado en su valor absoluto.

### Lee los valores
Usando input, agrega el código para leer la distancia del sol para cada planeta, considerando 2 planetas.

In [11]:
# Almacenar las entradas del usuario

Planeta1 = int(input("Dame la distancia del planeta 1 del sol: "))
Planeta2 = int(input("DAme la distancia del planeta 2 del sol: "))

### Convertir a numero
Debido a que input devuelve valores de cadena, necesitamos convertirlos en números. Para nuestro ejemplo, usaremos int

In [9]:
# Nota: Se hizo esto mismo en la linea de codigo anterior

### Realizar el calculo y convertir a valor absoluto
Con los valores almacenados como números, ahora puedes agregar el código para realizar el cálculo, restando el primer planeta del segundo. Debido a que el segundo planeta podría ser un número mayor, usarás abs para convertirlo a un valor absoluto. También agregarás el código para mostrar el resultado en millas multiplicando la distancia del kilómetro por 0.621

In [12]:
# Realizar el calculo y determinar el valor absoluto
Distancia = abs(Planeta2 - Planeta1)

print("La distancia entre los planetas en kilometros es: "+ str(Distancia))
print("La distancia entre los planetas en millas es: "+ str(Distancia*0.621))

La distancia entre los planetas en kilometros es: 50300000
La distancia entre los planetas en millas es: 31236300.0
